app/Http/Controllers/PhotoController.php

```diff
@@ -84,6 +90,20 @@ public function get(Request $request)
 
 		$return = $photo->prepareData();
 		$this->symLinkFunctions->getUrl($photo, $return);
+		if (!$this->sessionFunctions->is_logged_in()) {
```

Comment:
I find it suspicious that this functionality is guarded by `is_logged_in`. Haven't we recently fixed many instances of that? Being logged in shouldn't be enough to disable this -- the logged in user should be either the admin or the owner of the album...

Reply:
Hooo right nicely spotted.
I guess I need to use `is current user` then. :)